packages/webpack-scaffold/src/index.ts

```diff
@@ -109,7 +109,7 @@ export function InputValidate(
     message: string,
     cb?: (input: string) => string | boolean,
     defaultChoice?: string,
-    skip?: boolean,
+    skip = false,
```

Comment:
??

Reply:
To have a consistent style, if you take a look at other `prompts` we've an initial value for the `skip` argument. Also, there is type inference in place.

Reply:
Consistent style doesn't mean it's correct, you can give default fn params while maintaining types, please do that.

Reply:
In all other instances too

Reply:
Yeah that would capture type mismatch errors AOT.